src/main/java/org/killbill/billing/plugin/notification/setup/EmailNotificationListener.java

```diff
@@ -193,11 +204,12 @@ private void sendEmailForCancelledSubscription(final Account account, final ExtB
         final UUID subscriptionId = killbillEvent.getObjectId();
 
         final Subscription subscription = osgiKillbillAPI.getSubscriptionApi().getSubscriptionForEntitlementId(subscriptionId, context);
-        if (subscription != null && !muteEmailForObject(subscription.getBundleId(), ObjectType.BUNDLE, context)) {
-            final EmailContent emailContent = subscription.getState() == Entitlement.EntitlementState.CANCELLED ?
-                    templateRenderer.generateEmailForSubscriptionCancellationEffective(account, subscription, context) :
-                    templateRenderer.generateEmailForSubscriptionCancellationRequested(account, subscription, context);
-            sendEmail(account, emailContent, context);
+        if (subscription != null ) { //&& !muteEmailForObject(subscription.getBundleId(), ObjectType.BUNDLE, context)) {
+            //final EmailContent emailContent = subscription.getState() == Entitlement.EntitlementState.CANCELLED ?
+              //      templateRenderer.generateEmailForSubscriptionCancellationEffective(account, subscription, context) :
+               //     templateRenderer.generateEmailForSubscriptionCancellationRequested(account, subscription, context);
+            //sendEmail(account, emailContent, context);
+            subscriptionClient.sendEmailRequest("Subscription_Canceled", account, subscriptionId.toString());
```

Comment:
do we need to differentiate between Effective and Requested ?

Reply:
That is a good question I'm not sure what the difference is.